app/pages/sections/call-off-ordering-party/controller.js

```diff
@@ -3,14 +3,29 @@ import { getContext } from './contextCreator';
 import { getEndpoint } from '../../../endpoints';
 import { logger } from '../../../logger';
 
-export const getCallOffOrderingPartyContext = async ({ orderId, accessToken }) => {
-  const callOffOrgDataEndpoint = getEndpoint({ endpointLocator: 'getCallOffOrderingParty', options: { orderId } });
-  const callOffOrgData = await getData({ endpoint: callOffOrgDataEndpoint, accessToken, logger });
+export const getCallOffOrderingPartyContext = async ({ orderId, orgId, accessToken }) => {
+  try {
+    const callOffOrgDataEndpoint = getEndpoint({ endpointLocator: 'getCallOffOrderingParty', options: { orderId } });
+    const callOffOrgData = await getData({ endpoint: callOffOrgDataEndpoint, accessToken, logger });
 
-  if (callOffOrgData) {
-    logger.info(`Call off ordering party found in ORDAPI for ${orderId}: ${callOffOrgData}`);
-    return getContext({ orderId, data: callOffOrgData });
+    if (callOffOrgData) {
+      logger.info(`Call off ordering party found in ORDAPI for ${orderId}: ${callOffOrgData}`);
+      return getContext({ orderId, data: callOffOrgData });
+    }
+    logger.error(`No call off ordering party data returned from ORDAPI for ${orderId}`);
```

Comment:
should we be logging this as `error` it might be perfectly fine behaviour for it not to be there

Reply:
I'll remove it as I don't think it should ever be reachable anyway. If not found the api will send error 404